docs/source/release_notes/release-notes.rst

```diff
@@ -82,6 +82,16 @@ develop
            Instead, the exception is logged once only, when we run out of retries.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2432>`__)
 
+    *    - |new|
+         - AtlasDB clients now emit metrics that track the immutable timestamp, unreadable timestamp as well as current timestamp.
```

Comment:
nit: `,` --> `and`, xor `as well as` --> `, and`.

Reply:
Yep. Went with the second approach, `A and B as well as C` is correct but seemed unnecessary.

Reply:
also noticed the Oxford comma 😃